### PR TITLE
feat(ena-deposition): Small tweaks to uploaded metadata in manifest and project

### LINKED ISSUE #2977: Improve project deposition mapping/structure
Some ideas for improving the project submissions are in this thread: https://loculus.slack.com/archives/C0757PTR607/p1728464162216809

Currently we show like this:
image

We could try to get something more like this:
image

Showing like this in NCBI: 
image
Corresponding ENA project: https://www.ebi.ac.uk/ena/browser/view/PRJEB80643

The first sample we submitted was incorrectly labelled as genomic DNA - this has been fixed by submitting an EMBL file.

### PR DESCRIPTION
**Pull request by anna-parker**
resolves #2977, relates to #2983

preview URL: https://project-cleanup.loculus.org/

### Summary

1. Improve project by adding center name to project name and title (name=link and title=right below link of project shown in picture below)
image

2. Clean up address in manifest to also include center name
3. Improve error logging by printing contents of all files

### Screenshot

New projects will have this name/title/description

Name: `West Nile Virus: Genome sequencing by Test institution`
Title: `West Nile Virus: Genome sequencing by Test institution, city, country`
Description: `Automated upload of West Nile Virus sequences submitted by Test institution, city, country to Loculus`

### PR Checklist

- [ ] All necessary documentation has been adapted.
- [ ] The implemented feature is covered by an appropriate test.

### CODE REVIEW

File: ena-submission/Snakefile [hunk 1]
```diff
@@ -28,7 +28,7 @@ if SUBMIT_TO_ENA_DEV:
     print("Submitting to ENA dev environment")
     config["ena_submission_url"] = "https://wwwdev.ebi.ac.uk/ena/submit/drop-box/submit"
     config["github_url"] = (
-        "https://raw.githubusercontent.com/pathoplexus/ena-submission/main/test/approved_ena_submission_list.json"
+        "https://raw.githubusercontent.com/pathoplexus/ena-submission/loculus_test/test/approved_ena_submission_list.json"
```
Comment (anna-parker) [inline ena-submission/Snakefile:31]: revert before submitting

```suggestion
        "https://raw.githubusercontent.com/pathoplexus/ena-submission/main/test/approved_ena_submission_list.json"
```

File: ena-submission/scripts/create_assembly.py [hunk 2]
```diff
@@ -140,9 +140,18 @@ def create_manifest_object(
         try:
             group_info = get_group_info(config, project_table_entry["group_id"])[0]["group"]
             address = group_info["address"]
-            address_string = (f'{address.get("line1", "")}, {address.get("line2", "")}, '
-                f'{address.get("city", "")}, {address.get("state", "")}, '
-                f'{address.get("postalCode", "")}, {address.get("country")}')
+            address_list = (
+                [
+                    project_table_entry["center_name"],
```
Comment (corneliusroemer) [inline ena-submission/scripts/create_assembly.py:145]:
```suggestion
                    project_table_entry["center_name"],  # corresponds to Loculus' "Institution" group field
```

File: ena-submission/scripts/create_assembly.py [hunk 3]
```diff
@@ -140,9 +140,18 @@ def create_manifest_object(
...
+                    address.get("line1"),
+                    address.get("line2"),
```
Comment (corneliusroemer) [inline ena-submission/scripts/create_assembly.py:147]: I'd probably remove line1, line2, and post code. To keep it more concise. People can google the street

File: ena-submission/scripts/create_project.py [hunk 4]
```diff
@@ -92,8 +92,8 @@ def construct_project_set_object(
     project_type = ProjectType(
         center_name=XmlAttribute(group_info["institution"]),
         alias=alias,
-        name=metadata_dict["scientific_name"],
-        title=f"{metadata_dict["scientific_name"]}: Genome sequencing",
+        name=f"{metadata_dict["scientific_name"]}: Genome sequencing by {group_info["institution"]}",
+        title=f"{metadata_dict["scientific_name"]}: Genome sequencing by {group_info["institution"]}",
         description=(
             f"Automated upload of {metadata_dict["scientific_name"]} sequences submitted by {group_info["institution"]} from {config.db_name}",  # noqa: E501
```
Comment (corneliusroemer) [inline ena-submission/scripts/create_project.py:98]:
```suggestion
            f"Automated upload of {metadata_dict["scientific_name"]} sequences originally submitted by {group_info["institution"]} to {config.db_name}",  # noqa: E501
```

File: ena-submission/test/text_project_request.xml [hunk 5]
```diff
@@ -1,7 +1,7 @@
 <PROJECT_SET>
     <PROJECT center_name="Test institution" alias="1:Test organism:Test suffix">
-        <NAME>Test scientific name</NAME>
-        <TITLE>Test scientific name: Genome sequencing</TITLE>
+        <NAME>Test scientific name: Genome sequencing by Test institution</NAME>
+        <TITLE>Test scientific name: Genome sequencing by Test institution</TITLE>
```
Comment (corneliusroemer) [inline ena-submission/test/text_project_request.xml:4]: Could we actually put different things in here? Is that where the subtitles come from?

Like here:
image


It looks like it! We could put different information there then to make it look nicer

Brave Browser 2024-10-11 12 52 24

**Comment by corneliusroemer** [comment]
See comments here for how I could imagine the "Name"/"Title"/"Description" to be more informative and of increasing information content/length.

File: ena-submission/test/text_project_request.xml (hunk 5 above)
Comment (anna-parker) [reply ena-submission/test/text_project_request.xml]: I merged your changes :-)

**Review by corneliusroemer — APPROVED** [review]
Very cool, thanks!

In the future I could imagine also:
a) Adding the actual group name, to be able to get something like "Grubaugh Lab" in, and not just have Yale there, as it's useful to get as specific as possible
b) Potentially use "brokered upload" instead of "automated upload" and "originally submitted to Loculus" instead of "submitted to Loculus" but none of this is blocking, we can discuss in future

**Comment by anna-parker** [comment]
Added group name as well now

**Pull request merged**